diff --git a/src/CVCard.ts b/src/CVCard.ts
index 793e095..8e09a6b 100644
--- a/src/CVCard.ts
+++ b/src/CVCard.ts
@@ -1,14 +1,10 @@
 import { html, TemplateResult } from "lit-html";
-import { icons } from "solid-ui";
-
-const editButtonURI = icons.iconBase + 'noun_253504.svg'
-
 import {
   fullWidth,
   heading,
   paddingSmall,
   textCenter,
-  textLeft, textRight,
+  textLeft,
   textGray,
 } from "./baseStyles";
 import { ProfilePresentation } from "./presenter";
@@ -21,7 +17,6 @@ const styles = {
   intro: styleMap({ ...textGray(), ...textCenter() }),
   card: styleMap(card()),
   info: styleMap({ ...paddingSmall(), ...textLeft() }),
-  tools: styleMap({ ...paddingSmall(), ...textRight() }),
 };
 
 export const CVCard = (
@@ -52,7 +47,6 @@ export const CVCard = (
       <div style=${styles.info}>${renderSkills(skills)}</div>
       <h3 style=${nameStyle}>Languages</h3>
       <div style=${styles.info}>${renderLanguages(languages)}</div>
-      <div style=${styles.tools}>${renderEditButton()}</div>
     </div>
     </div>
   `}
@@ -111,9 +105,3 @@ function strToUpperCase(str) {
   }
   return '';
 }
-
-function renderEditButton () {
-  return html `<button type="button" class="ProfilePaneCVEditButton">
-  <img  src="${editButtonURI}">
-  Edit</button>`
-}
\ No newline at end of file
